dev/mcal/src/Rcc.cpp

```diff
@@ -70,6 +69,14 @@ void Rcc::InitSysClock(const ClkConfig& config,
     RCC->CFGR.SW = 2;    // Switch to PLL
 }
 
+void Rcc::SetExternalCrystal() {
```

Comment:
The purpose of  this function is to configure the HSE either to a crystal/ceramic oscillator ( Not bypassed) or RC (bypassed)
To achieve this, you can add a parameter to the `ConfigureExternalClock(HSE HSE_Type)` function that allows you to choose the HSE type.